src/Hostpinnacle.php

```
@@ -118,40 +110,36 @@ private function setRequestOptions()
         if (is_null($method)) {
             throw new IsNullException('Method must not be null');
         }
 
-        $response = $this->client->{strtolower($method)}(
+        return $this->client->{strtolower($method)}(
             $this->baseUrl . $relativeUrl,
             ["body" => json_encode($body)]
         );
-
-        return $response;
     }
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'setHttpResponse'. (undefined)

(UnusedPrivateMethod)